frontend/src/components/Admin/AdminShoutouts/AdminShoutouts.tsx

```diff
@@ -21,31 +23,165 @@ const AdminShoutouts: React.FC = () => {
             'Please make sure the latest shoutout date is after the earliest shoutout date.'
         });
       } else {
-        setDisplayShoutouts(
-          shoutouts.filter((shoutout) => {
+        const filteredShoutouts = shoutouts
+          .filter((shoutout) => {
             const shoutoutDate = new Date(shoutout.timestamp);
-            return !shoutout.hidden && shoutoutDate >= earlyDate && shoutoutDate <= lastDate;
+            return shoutoutDate >= earlyDate && shoutoutDate <= lastDate;
           })
-        );
+          .sort((a, b) => a.timestamp - b.timestamp);
+        setAllShoutouts(filteredShoutouts);
+        if (view === 'PRESENT')
+          setDisplayShoutouts(filteredShoutouts.filter((shoutout) => !shoutout.hidden));
+        else if (view === 'HIDDEN')
```

Comment:
Can we make something like a `ViewMode` type to explicit define the options for the view ('ALL', 'PRESENT', 'HIDDEN')? I think itll be more clear to other people working on this to define all of them in one place and potentially avoid some weird bugs in the future from allowing arbitrary strings for `view`.